features/dd-sdk-android-rum/src/test/kotlin/com/datadog/android/rum/internal/domain/scope/RumViewScopeTest.kt

```diff
@@ -485,6 +486,7 @@ internal class RumViewScopeTest {
             type = expectedViewType,
             trackFrustrations = fakeTrackFrustrations
         )
+        reset(mockInternalLogger)
```

Comment:
why do we need a `reset` call here as well?

Reply:
Same thing, some flakiness rose up during the setup